docs/guides/capabilities/ai/meeting-transcription.mdx

```diff
@@ -0,0 +1,53 @@
+This guide will walk you through how to use this feature effectively.
+
+## Control transcriptions for participants using presets
+
+You can control whether or not a participant's audio will be transcribe with the help of the `transcription_enabled` flag in the participant's preset.
+All participants with the `transcription_enabled` turned on in their preset will be able to generate transcripts in real-time in a Dyte meeting.
+
+You can create a new preset on our [Developer Portal](https://dev.dyte.io/presets), or using our [REST API](https://docs.dyte.io/api#/operations/post-presets)
+
+## Consuming transcripts
+
+There are 2 ways in which these transcripts can be consumed.
+1. The transcripts can be consumed on the client-side using the Dyte SDK that's suitable for your platform. These transcripts are generated on the server in real-time.
+2. The meeting transcript can be consumed via a [webhook after the meeting ends](#consume-transcript-via-a-post-meeting-webhook).
+
+### Consuming transcripts in real-time
+
+For consuming transcripts in real-time on the client SDK of your choice, you just need to ensure that the `transcription_enabled` flag is enabled in the preset.
+Transcripts for all the participants having this flag set will be broadcasted in the meeting.
+
+### Consume transcript via a post-meeting webhook
+
+You can configure a webhook with the `meeting.transcript` event enabled to receive the meeting transcript after the meeting has ended.
+You can do this either on our [Developer Portal](https://dev.dyte.io/webhooks), or using a [REST API](https://docs.dyte.io/api#/operations/addWebhook).
+
+## Fetch the meeting transcript
+
+You do not need to rely on the webhook to get the transcript for a meeting. Dyte provides a [REST API](https://docs.dyte.io/api#/operations/GetSessionTranscript) using which you can obtain the transcripts for a particular session.
+You can use this API to get the transcript for a meeting at a later time. Dyte stores the transcript of a meeting for 7 days since the start of the meeting.
+
+The transcript is received in the form of a CSV. Here is the format of the said CSV:
+
+```csv
+Timestamp, Participant ID, Participant Name, Transcript
+```
+
+## Testing transcription
+
+Once you have configured a preset and a webhook according to the instructions above, you can proceed to test whether meeting transcription is working for your organization.
+To test if meeting transcription has been configured for your organization, perform the following steps.
+
+1. [Create a meeting](https://docs.dyte.io/api#/operations/create_meeting).
+2. [Add a participant](https://docs.dyte.io/api#/operations/add_participant) to the meeting. Make sure that the preset you use was configured according to this guide.
+3. Join the meeting with the `authToken` you just obtained. As you unmute and speak (in English), your speech should be getting transcribed in real-time for all the participants in the meeting.
```

Comment:
Could we add an attention/information box in the beginning that says that Dyte's transcription currently only supports English?

Reply:
![image](https://github.com/dyte-in/docs/assets/42958812/df38cec6-54e1-4b64-b7e2-79c73160677f)

Added this note after the introductory paragraph.